bokeh/embed.py

```diff
@@ -63,19 +64,43 @@ def components(plot_objects, resources=None):
 
         resources : Deprecated argument
 
+        wrap_script (boolean, optional): If set to false, the returned javascript is not wrapped
+        in a script tag. (default: True)
+
+        wrap_plot_info (boolean, optional): If true, then a set of divs are returned as described
+        above. If set to false, then dictionaries are returned that can be used to manually
+        build your own divs. The dictionary contains following information:
+        {
+            'modelid':  'The plots id, which can be used in the Bokeh.index',
+            'elementid': 'The css identifier the BokehJS will look for to target the plot',
+            'modeltype': 'Plot',
+        }
+        (default: True)
```

Comment:
Docstrings can be multi-part, I like to put the default closer to the beginning:

```
        wrap_plot_info (boolean, optional): If true, returns divs as described above . If False, 
        then returns dictionaries that can be used to manually build your own divs. (default: True)

        Returned dictionaries contain the following information:
        {
            'modelid':  'The plots id, which can be used in the Bokeh.index',
            'elementid': 'The css identifier the BokehJS will look for to target the plot',
            'modeltype': 'Plot',
        }
```
Might need to check the format, it's possible the lines underneath have to be indented, I can't remember offhand.